dnf-behave-tests/features/steps/common/file.py

```diff
@@ -16,6 +51,10 @@ def ensure_directory_exists(dirname):
 def ensure_file_exists(filename):
     assert os.path.exists(filename), "ENSURE: file exists {!r}".format(filename)
 
+def regex_ensure_file_exists(filename):
```

Comment:
I'd rather do
```
f = regex_find_file(filename)
ensure_file_exists(f)
```
instead of creating this new function. It is only called from one place.